config/webpack/alias-builder.js

```diff
@@ -37,12 +37,20 @@ function setSpaAlias(alias, moduleName, filePath) {
 }
 
 module.exports = {
-  buildAliasList: function () {
+  buildAliasList: function (skyPagesConfig) {
```

Comment:
Just confirming, we were already passing in `skyPagesConfig` when this method is called, we just weren't using it here?

Reply:
That's correct.